examples/wgpu/wgpu_instancing/shaders/shader.vert

```diff
@@ -10,7 +10,11 @@ layout(location = 0) in vec3 position;
 layout(location = 1) in vec3 normal;
 //locations 3,4,5, are implicitely taken by `instance_transform`
 //since one location can contain at max vec4
-layout(location = 2) in mat4 instance_transform;
+// layout(location = 2) in mat4 instance_transform;
```

Comment:
Would you mind leaving a note that mentions this is a temporary workaround for a wgpu limitation? thanks!

Reply:
Hmm, but is it temporary? The tutorial says it should be like this: https://sotrh.github.io/learn-wgpu/beginner/tutorial7-instancing/#the-instance-buffer

Reply:
I should remove that line regardless :D

Reply:
I've added a comment